DOCS/man/options.rst

```diff
@@ -738,10 +738,12 @@ Video
     choices if you encounter video that has the wrong aspect ratio in mpv,
     but seems to be correct in other players.
 
-``--video-unscaled``
+``--video-unscaled=<no|yes|downscale-big>``
     Disable scaling of the video. If the window is larger than the video,
-    black bars are added. Otherwise, the video is cropped. The video still
-    can be influenced by the other ``--video-...`` options.
+    black bars are added. Otherwise, the video is cropped, unless the option
+    is set to ``downscale-big``, in which case the video is fit to window as
+    usual. The video still can be influenced by the other ``--video-...``
```

Comment:
I haven't been able to figure what `vo_panscan_area` is for, I left the assignment as it was introduced in b968d779